time/src/main/java/net/jqwik/time/internal/properties/arbitraries/DefaultOffsetTimeArbitrary.java

```diff
@@ -0,0 +1,180 @@
+package net.jqwik.time.internal.properties.arbitraries;
+
+import java.time.*;
+import java.time.temporal.*;
+
+import org.apiguardian.api.*;
+
+import net.jqwik.api.*;
+import net.jqwik.api.arbitraries.*;
+import net.jqwik.time.api.*;
+import net.jqwik.time.api.arbitraries.*;
+
+import static java.time.temporal.ChronoUnit.*;
+import static org.apiguardian.api.API.Status.*;
+
+@API(status = INTERNAL)
+public class DefaultOffsetTimeArbitrary extends ArbitraryDecorator<OffsetTime> implements OffsetTimeArbitrary {
+
+	private LocalTime timeMin = LocalTime.MIN;
```

Comment:
I still think that delegating all configuration to an embedded localTimeArbitrary and zoneIdArbitrary could save a lot of code.

Reply:
How can I embed them? What exactly do you mean? Do you have an example?

Reply:
The simplest example is probably DefaultBigIntegerArbitrary. The idea is that you delegate the configuration, e.g. between(min, max), to the embeded arbitrary:
```java
OffsetTimeArbitrary clone = this.cloneTyped();
clone.localTimeArbitrary = localTimeArbitrary.between(min, max);
return clone;
```

Reply:
Done.